docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_with_a_profiler.md

```diff
@@ -31,101 +31,88 @@ These Expectations are deliberately over-fitted on your data e.g. if your table
+Run the following command to identify the <TechnicalTag tag="batch" text="Batch" /> of data you want to <TechnicalTag tag="profiling" text="Profile" /> to create your Expectation Suite:
 
 ```python name="tests/integration/docusaurus/expectations/how_to_create_and_edit_expectations_with_a_profiler name_suite"
 ```
 
-### 3. Instantiate your Validator
+## 3. Instantiate your Validator
 
-We use a Validator to access and interact with your data. We will be passing the Validator to our Profiler in the next step.
+Run the following command to use a <TechnicalTag tag="validator" text="Validator" /> to access and interact with your data:
 
 ```python name="tests/integration/docusaurus/expectations/how_to_create_and_edit_expectations_with_a_profiler create_validator"
 ```
 
-After you get your Validator, you can call `validator.head()` to confirm that it contains the data that you expect.
+Run `validator.head()` to confirm that it contains the expected data.
 
-### 4. Instantiate a UserConfigurableProfiler
+## 4. Instantiate a UserConfigurableProfiler
 
-Next, we instantiate a UserConfigurableProfiler, passing in the Validator with our data:
+Run the following command to instantiate a UserConfigurableProfiler and pass in the Validator with your data:
 
 ```python name="tests/integration/docusaurus/expectations/how_to_create_and_edit_expectations_with_a_profiler create_profiler"
 ```
 
-### 5. Use the profiler to build a suite
+## 5. Use the profiler to build a suite
 
-Once we have our Profiler set up with our Batch, we call `profiler.build_suite()`. 
-This will print a list of all the Expectations created by column, and return the Expectation Suite object.
+Run the following command to print a list of all the Expectations created by column, and return the Expectation Suite object:
 
 ```python name="tests/integration/docusaurus/expectations/how_to_create_and_edit_expectations_with_a_profiler build_suite"
 ```
 
-### 6. (Optional) Running validation, saving your suite, and building Data Docs
+## 6. Running validation, saving your suite, and building Data Docs (Optional)
 
-If you'd like, you can <TechnicalTag tag="validation" text="Validate" /> your data with the new Expectation Suite, 
-save your Expectation Suite, and build <TechnicalTag tag="data_docs" text="Data Docs" /> to take a closer look 
-at the output:
+Run the following command to <TechnicalTag tag="validation" text="Validate" /> your data with the new Expectation Suite, save your Expectation Suite, and build the <TechnicalTag tag="data_docs" text="Data Docs" />:
 
 ```python name="tests/integration/docusaurus/expectations/how_to_create_and_edit_expectations_with_a_profiler e2e"
 ```
 
-And you're all set!
-
 ## Optional Parameters
 
-The UserConfigurableProfiler can take a few different parameters to further hone the results. These parameters are:
+The following are the optional parameters for UserConfigurableProfiler:
```

Comment:
I feel like this is something we could handle by linking to the API doc for the `UserConfigurableProfiler`, which would ensure completeness and automatic updates if the list changes in the future (since the docstring that generates the API doc page would be updated).

Side note: generally speaking, any proper class names or parameter names should be wrapped in back ticks.  So: `UserConfigurableProfiler` vs User Configurable Profiler.

Reply:
@Rachel-Reverie as recommended, I removed the text and bulleted list and linked to the API reference.